src/panel/NoResultPanel.jsx

```diff
@@ -0,0 +1,36 @@
+/* globals _ */
+import React from 'react';
+
+import { Panel } from 'src/components/ui';
+
+const close = () => window.app.navigateTo('/');
+
+const handleSearchClick = e => {
+  e.preventDefault();
+  close();
+  window.__searchInput.searchInputHandle.value = '';
```

Comment:
To be tested, but you should be able to avoid this global stuff manip by importing `SearchInput` and calling `setInputValue` then `select`.

Reply:
I confirm it works! thansk